### DISCUSSION TITLE
Postgres extensions

### DISCUSSION THREAD

**Discussion by tim-vandecasteele** [ubicloud/ubicloud#2467] [General]
Hi, managed postgres on Hetzner is a great proposition. I was wondering what the policy is on upgrading extensions? I'm mainly asking because I'm looking to have pgvector 0.0.8. If I understand [this page](https://www.ubicloud.com/docs/managed-postgresql/extensions) correctly, currently pgvector 0.0.6 is installed? (although it is listed as 'vector'). 
What's your policy of keeping extensions up to date?

**Comment by byucesoy** [comment]
Hi @tim-vandecasteele,
It seems the documentation is outdated. Current active version of pg_vector is `0.7.4`. I also just started to update pg_vector to version `0.8.0`. If there is any other blocker for you to try Ubicloud PostgreSQL, just let me know, we would be happy to assist.
